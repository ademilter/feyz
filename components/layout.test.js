import { render, screen } from '@testing-library/react'
import Layout from './layout'

describe('Layout', () => {
  test('should have children.', () => {
    const child = '<span>Test Message</span>'

    const { asFragment } = render(<Layout>{child}</Layout>)

    expect(asFragment()).toMatchSnapshot()
    expect(screen.getByText(/Test Message/i)).not.toBeNull()
  })
})
