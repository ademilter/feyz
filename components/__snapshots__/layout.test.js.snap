// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should have children. 1`] = `
<DocumentFragment>
  <div
    class=""
  >
    &lt;span&gt;Test Message&lt;/span&gt;
  </div>
</DocumentFragment>
`;
